packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -129,13 +129,7 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
     }
 
     public async uploadSummaryWithContext(summary: ISummaryTree, context: ISummaryContext): Promise<string> {
-        const summaryHandle = await PerformanceEvent.timedExecAsync(
-            this.logger,
-            {
-                eventName: "uploadSummaryWithContext",
-            },
-            async () => this.summaryUploadManager.writeSummaryTree(summary, context.ackHandle ?? "", "channel"),
-        );
+        const summaryHandle =  this.summaryUploadManager.writeSummaryTree(summary, context.ackHandle ?? "", "channel");
```

Comment:
I don't think we should remove this event from the routerlicious-driver since it doesn't have the uploadSummary event.

Reply:
@vladsud since routerlicious has uploadSummaryWithContext events, should we remove "uploadSummary" event instead, to keep it consistent,
or rename these in routerlicious to "uploadSummary"?